content/departments/product-engineering/engineering/process/deployments/playbooks.md

```diff
@@ -64,6 +64,8 @@ More about [pipeline run types](https://docs.sourcegraph.com/dev/background-info
 
 **Deploy**
 
+<!--  TODO: line 69: should we remove CloudSaas and keep just CLoudDevops? -->
+
```

Comment:
Wondering if we can/should remove Cloud SaaS and keep just Cloud Devops or maybe Cloud Org?

Reply:
Yes - Lets simply remove Cloud SaaS. DevOps probably should own this anyways.